README.md

```diff
@@ -12,6 +12,15 @@ This package uses the [`remembrance-agent` engine package](https://github.com/re
 
 ![Logo](./docs/img/logo.png)
 
+## Installation Requirements
+
+### Debian-Based Systems (Ubuntu)
+
+System Packages:
+
+- openjdk-8-jdk
```

Comment:
Good suggestion! I definitely need to spend time documenting the requirements for this package. Do you know of a way to make the desktop client portable? It'd be great if we can just deploy it for different platforms as a self-contained executable